api/app/routers/land_change/dist_alerts/dist_alerts.py

```diff
@@ -0,0 +1,122 @@
+import json
+import uuid
+from pathlib import Path
+
+from fastapi import APIRouter, Request, BackgroundTasks, HTTPException
+from fastapi import Response as FastAPIResponse
+from fastapi.responses import ORJSONResponse
+
+from app.analysis.dist_alerts.analysis import do_analytics
+
+from app.models.common.analysis import AnalysisStatus
+from app.models.common.base import (
+    DataMartResourceLinkResponse,
+    DataMartResourceLink,
+)
+from app.models.land_change.dist_alerts import (
+    DistAlertsAnalyticsResponse,
+    DistAlertsAnalyticsIn,
+)
+
+
+router = APIRouter(prefix="/dist_alerts")
+
+PAYLOAD_STORE_DIR = Path("/tmp/dist_alerts_analytics_payloads")
+PAYLOAD_STORE_DIR.mkdir(parents=True, exist_ok=True)
+
+
+@router.post(
+    "/analytics",
+    response_class=ORJSONResponse,
+    response_model=DataMartResourceLinkResponse,
+    status_code=202,
+)
+def create(
+    *, data: DistAlertsAnalyticsIn, request: Request, background_tasks: BackgroundTasks
+):
+    # Convert model to JSON with sorted keys
+    payload_dict = data.model_dump()
+
+    # Convert to JSON string with sorted keys
+    payload_json = json.dumps(payload_dict, sort_keys=True)
+
+    # Generate deterministic UUID from payload
+    resource_id = str(uuid.uuid5(uuid.NAMESPACE_DNS, payload_json))
+
+    # Store payload in /tmp directory
```

Comment:
I suppose this is also a part we might want to think about this sprint - we should save to S3 or a permanent file store!

Reply:
Think about for sure. But, i don't think we should try to implement it yet. I want to still refactor these routers.